js/file.js

```diff
@@ -0,0 +1,52 @@
+let books = JSON.parse(localStorage.getItem('cataloge'));
+const file = document.querySelector('.book');
+
+function addBooks(ti, aut) {
+  books.push({ title: ti, author: aut, id: books.length });
+  window.localStorage.setItem('cataloge', JSON.stringify(books));
+}
+
+/* eslint-disable */
+function removeBooks(index) {
+  books = books.filter((book) => book.id !== index);
+
+  const element = document.getElementById(index);
+  element.parentNode.removeChild(element);
+
+  window.localStorage.setItem('cataloge', JSON.stringify(books));
+}
+/* eslint-enable */
+
+function showBooks() {
+  for (let i = 0; i < books.length; i += 1) {
```

Comment:
**Your project isn't working well i can't display books.**
- Kindly fix errors displayed on chrome console related to line 21 of this file.